src/common/utxobased/network/BlockBookAPI.ts

```diff
@@ -0,0 +1,87 @@
+import { EdgeTransaction } from 'edge-core-js'
+
+export interface PartialTask {
+  method: string
+  params: unknown
```

Comment:
Add cleaner and use it on return task

Reply:
Also add error message to use when cleaner failed or rejected.

Reply:
Can this be done in a later task? Seems to only be tangentially related to pick next task. The current code also has no cleaners.